compiler/rustc_codegen_ssa/src/target_features.rs

```diff
@@ -236,15 +233,14 @@ pub fn cfg_target_feature(
             if RUSTC_SPECIAL_FEATURES.contains(feature) {
                 return true;
             }
-            is_feature_enabled(feature)
+            target_base_has_feature(feature)
```

Comment:
(pre-existing) why do we filter on this, shouldn't we error if the backend doesn't know a feature that was requested?

Reply:
But also, the line you put the comment on isn't even about the backend knowing features. This line here checks, for every feature we do know, whether it is enabled. I don't understand when you think that should error.